packages/itwinui-css/src/surface/surface.scss

```diff
@@ -22,5 +22,43 @@
 
 // Public facing surface component.
 @mixin iui-surface-component {
+  --iui-surface-padding-inline: var(--iui-size-s);
+  --iui-surface-body-padding: var(--iui-surface-padding-inline);
+
   @include iui-surface;
+  display: flex;
+  flex-direction: column;
+  height: var(--iui-surface-height);
```

Comment:
Unless we provide some preset values, this variable doesn't help, because the user can always specify `height` in their own CSS using a custom class.

Reply:
I was envisioning something like this:
```html
<wrapper style="--iui-surface-height: 100px;">
	<surface />
	<surface />
	<surface />
</wrapper>
```

I think you're proposing this:
```css
.myClass { height: 100px; }
```
```html
<wrapper>
	<surface class="myClass" />
	<surface class="myClass" />
	<surface class="myClass" />
</wrapper>
```
Do you still prefer your suggestion?

Reply:
yes, because I see no benefit of the CSS variable in this case.

Reply:
Removed the height, am setting the height using `grid-auto-rows` in the demo CSS.